app/components/post_add_channel_member/post_add_channel_member.js

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    defineMessages,
+    intlShape,
+} from 'react-intl';
+
+import {Text} from 'react-native';
+
+import {General} from 'mattermost-redux/constants';
+
+import {concatStyles} from 'app/utils/theme';
+
+import AtMention from 'app/components/at_mention';
+import FormattedText from 'app/components/formatted_text';
+
+export default class PostAddChannelMember extends React.PureComponent {
+    static propTypes = {
+        actions: PropTypes.shape({
+            addChannelMember: PropTypes.func.isRequired,
+            removePost: PropTypes.func.isRequired,
+            sendEphemeralPost: PropTypes.func.isRequired,
+        }).isRequired,
+        currentUser: PropTypes.object.isRequired,
+        channelType: PropTypes.string.isRequired,
```

Comment:
In the index you are passing this as empty in one of the cases so either change that or remove the required here

Reply:
You're right. I'll just remove required here.